### PR TITLE
add documentation for adding @yoast-components

### PR DESCRIPTION
**Pull request by GuidoOffermans**
## Summary

This PR can be summarized in the following changelog entry:

* Added documentation on how to add `@yoast-components` to your project.

## Quality assurance

* [x] I have tested this code to the best of my abilities
* [ ] I have added unittests to verify the code works as intended

Fixes #

### CODE REVIEW

File: packages/components/README.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+
+### Now the JavaScript has been taken care of, we need to make sure CSS will also load.
+
 Because we are importing CSS in our JavaScript, your JavaScript bundler needs to be able to interpret CSS.  
 
 Therefore, you will need to use a css-loader in your bundler in order to use this package.
 
 E.g. in Webpack: https://webpack.js.org/loaders/css-loader/
 
+Make sure to add the CSS imports to your project. `import "@yoast/components/base";`
```
Comment (Dieterrr) [inline packages/components/README.md:18]: This is a bit unclear to me: Where do I add this line? And what does it mean? I expect to import css but yoast/components/base sounds like something else.

File: packages/components/README.md [hunk 2]
```diff
@@ -1,10 +1,22 @@
 ## Requirements
+The Yoast/Components package is not pre-build. So if you would like to include `@yoast/components` into your project we will need a few steps to set it up.
+
```
Comment (Dieterrr) [inline packages/components/README.md:3]:
```suggestion
### Loading Javascript. 

```

File: packages/components/README.md [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
+1. Install the package by running `yarn add @yoast/components`.
+2. Include Yoast/components into the babel-loader of your `webpack.config`. This will look something like this: `include: [ paths.appSrc, /node_modules[/\\](yoast-components|@yoast)[/\\].*/ ]`
+3. It can be that you will need presets for the `babel-loader`. These are: `presets: [ "@babel/preset-env", "@babel/preset-react" ]`
+	
+	Note that you will also need to install these presets. Run `yarn add --dev @babel/preset-env @babel/preset-react`
+
+### Now the JavaScript has been taken care of, we need to make sure CSS will also load.
```
Comment (Dieterrr) [inline packages/components/README.md:10]:
```suggestion
### Loading CSS
```
Nothing wrong with the sentence, just a bit wordy for a header :)

File: packages/components/README.md (hunk 1 above)
Comment (GuidoOffermans) [reply packages/components/README.md:18]: You're right this could be clearer.

**Comment by Dieterrr** [comment]
CR done 👍
